xml/admin_security.xml

```diff
@@ -783,6 +783,223 @@ roleRef:
    </example>
   </sect2>
  </sect1>
+
+ <sect1 xml:id="sec.admin.security.pod_policies">
+  <title>Pod Security Policies</title>
+  <para>
+   This section provides an overview of policy settings for pod
+   security. By default, pod security policies are already enabled on
+   &productname;.
+  </para>
+  <para>
+   &productname; comes with 2 defined policies. The required role
+   definition is also included in the examples. Role bindings have
+   to be created manually. For details about roles and role bindings,
```

Comment:
> Role bindings have to be created manually.

Kinda. A default RoleBinding exists to give all authenticated users and service accounts permission to use the "suse:caasp:psp:unprivileged" PSP. Additional RoleBindings will be required for users/serviceaccounts which needs access to other PSPs.